apps/xen-tg-app/src/components/AnimatedBackground.css

```diff
@@ -34,34 +34,52 @@
   }
 }
 
+.noise {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  overflow: hidden;
+  top: 0;
+  left: 0;
+  background: var(--color-bg2);
+  background-image: url(@/assets/noise-dark.png);
+  background-blend-mode: screen;
```

Comment:
With these styles looks better

```css
background-size: 110px 110px;
filter: opacity(0.5);
```

Reply:
I agree with `background-size` but not the opacity filter. The goal was to make the background in the light theme lighter and in the dark theme darker. Animated spots interfere with meaningful blocks.